editor/editor_node.cpp

```diff
@@ -691,10 +691,12 @@ void EditorNode::_notification(int p_what) {
 
 			bool theme_changed =
 					EditorSettings::get_singleton()->check_changed_settings_in_group("interface/theme") ||
-					EditorSettings::get_singleton()->check_changed_settings_in_group("text_editor/theme") ||
 					EditorSettings::get_singleton()->check_changed_settings_in_group("interface/editor/font") ||
 					EditorSettings::get_singleton()->check_changed_settings_in_group("interface/editor/main_font") ||
+					EditorSettings::get_singleton()->check_changed_settings_in_group("interface/editor/main_font_bold") ||
```

Comment:
If `check_changed_settings_in_group` matches a prefix, this is already included in the previous line with `main_font`.

Reply:
Oh yeah, it does. I just got confused about it a few times checking this.